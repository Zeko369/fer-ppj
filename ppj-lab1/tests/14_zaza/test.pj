c = 0
za a od 0 do 5
  za b od a do 10
    c = c + a*b
  az
az
