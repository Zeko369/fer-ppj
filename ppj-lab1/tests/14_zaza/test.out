<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 c
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 0
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <za_petlja>
     KR_ZA 2 za
     IDN 2 a
     KR_OD 2 od
     <E>
      <T>
       <P>
        BROJ 2 0
       <T_lista>
        $
      <E_lista>
       $
     KR_DO 2 do
     <E>
      <T>
       <P>
        BROJ 2 5
       <T_lista>
        $
      <E_lista>
       $
     <lista_naredbi>
      <naredba>
       <za_petlja>
        KR_ZA 3 za
        IDN 3 b
        KR_OD 3 od
        <E>
         <T>
          <P>
           IDN 3 a
          <T_lista>
           $
         <E_lista>
          $
        KR_DO 3 do
        <E>
         <T>
          <P>
           BROJ 3 10
          <T_lista>
           $
         <E_lista>
          $
        <lista_naredbi>
         <naredba>
          <naredba_pridruzivanja>
           IDN 4 c
           OP_PRIDRUZI 4 =
           <E>
            <T>
             <P>
              IDN 4 c
             <T_lista>
              $
            <E_lista>
             OP_PLUS 4 +
             <E>
              <T>
               <P>
                IDN 4 a
               <T_lista>
                OP_PUTA 4 *
                <T>
                 <P>
                  IDN 4 b
                 <T_lista>
                  $
              <E_lista>
               $
         <lista_naredbi>
          $
        KR_AZ 5 az
      <lista_naredbi>
       $
     KR_AZ 6 az
   <lista_naredbi>
    $
